recipes/cap3/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: cap3
+  version: '10.2011'
+
+source:
+  fn: cap3.linux.x86_64.tar # [linux]
+  url: http://seq.cs.iastate.edu/CAP3/cap3.linux.x86_64.tar # [linux]  
```

Comment:
Can you try to compile it from source? Precompiled binaries are not guaranteed to run on older machines.